UnityProject/Assets/Standard Assets/HSVPicker/UI/ColorPicker.cs

```diff
@@ -195,6 +204,12 @@ private float A
         }
     }
 
+    public void EnablePicker(Action<Color> onChange)
+    {
+	    DynamicValueChangeEvent += onChange;
```

Comment:
Make sure to unsubscribe from this if the object is destroyed or when you finish interacting

Reply:
I'm already doing that